cmd/root.go

```diff
@@ -117,9 +118,43 @@ func MakeCommands() *cobra.Command {
 	// just the error message.
 	rootCmd.SilenceUsage = true
 
+	setFlagErrorFuncAndValidateArgs(rootCmd)
+
 	return rootCmd
 }
 
+func setFlagErrorFunc(cmd *cobra.Command, err error) error {
+	if e := cmd.Help(); e != nil {
+		return e
+	}
+	fmt.Println("")
```

Comment:
?

Reply:
These newlines are here to add a visual break between the help text and the error message.

Please see the PR description for an example :bow: